src/main/blackbox/blackbox_io.c

```diff
@@ -470,7 +470,7 @@ bool blackboxDeviceOpen(void)
      *
      * 9 / 1250 = 7200 / 1000000
      */
-    blackboxWriteChunkSize = MAX((masterConfig.looptime * 9) / 1250, 4);
+    blackboxWriteChunkSize = MAX((GUARD_LOOPTIME * 9) / 1250, 4);
```

Comment:
I'll make the heuristic overridable at least for same time, so that new users start with heuristic, but version upgrade will be closer to old behavior (with looptime rounded up to next available gyro sampling rate).

Rule of thumb is that sampling time should ve 10x-100x system time constant, so there is no gain going over 1kHz looptime (but I may be wrong).
Personally I place looptime into 250Hz range now, but it may be better to go a bit higher..
I'am more interested in 'advanced' algorithms that will allow higher looptime then brute-forcing it .. 

BTW: attitude is now estimated at loop frequency, but ACC correction is filtered at 1/600. 
This is probably overkill, ACC correction may be done at much lower frequency (maybe using filtered ACC data) and just integrate gyro reading on each loop (and integration may be even improved if low-noise gyro rate derivative derivative is available). This will greatly reduce ACC penalty ...